ios/Fabric/RNCPagerViewComponentView.mm

```diff
@@ -83,6 +84,27 @@ - (void)willMoveToSuperview:(UIView *)newSuperview {
     }
 }
 
+-(void) didMoveToWindow {
+    UIPanGestureRecognizer* gesture = [UIPanGestureRecognizer new];
```

Comment:
[high]: Could you add this behaviour conditionally based on a prop, which comes from the react native component ?